data/human/human missions.txt

```diff
@@ -5391,7 +5391,7 @@ mission "Deal Change 1"
 mission "Deal Change 2"
 	landing
 	name "Inheritance to <planet>"
-	description "Instead of bringing the inheritance to Skymoot, deliver it to <destination>. You were promised <payment> for doing so."
+	description "Instead of bringing the inheritance to Skymoot, deliver it to <destination>. You were promised <payment> for doing so. To bypass <planet>'s spaceport toll, hail <planet> and read out the code to the spaceport authority."
```

Comment:
```suggestion
	description "Instead of bringing the inheritance to Skymoot, deliver it to <destination>. You were promised <payment> for doing so. To bypass the spaceport toll, hail <destination> and read out the code to the spaceport authority."
```

No need to repeat the planet name so often.

Also, I think the added substitution may be wrong (pointing back at Skymoot instead), but I'm honestly already a bit confused by its continued usage in the mission name while directly spelling out 'Skymoot' in this bit of the text.

Reply:
Skymoot is not involved in the mission; it's where the original inheritance mission leads to.

Reply:
Fair enough – but I'll replace `<destination>` with `<planet>` because now `Greenrock in the Shaula system` will be repeated.